packages/eslint-plugin/src/util/index.ts

```diff
@@ -36,16 +36,15 @@ const {
   nullThrows,
   NullThrowsReasons,
 } = ESLintUtils;
-type InferMessageIdsTypeFromRule<T> =
+export type InferMessageIdsTypeFromRule<T> =
   ESLintUtils.InferMessageIdsTypeFromRule<T>;
-type InferOptionsTypeFromRule<T> = ESLintUtils.InferOptionsTypeFromRule<T>;
+export type InferOptionsTypeFromRule<T> =
+  ESLintUtils.InferOptionsTypeFromRule<T>;
 
 export {
   applyDefault,
   deepMerge,
   getParserServices,
-  type InferMessageIdsTypeFromRule,
-  type InferOptionsTypeFromRule,
   isObjectNotArray,
   nullThrows,
   NullThrowsReasons,
```

Comment:
You could export the const above instead of exporting here, couldn't you?

Reply:
True!